spec/mongoid/scopable_spec.rb

```diff
@@ -110,6 +110,21 @@
         expect(Band).to be_default_scoping
       end
     end
+    
+    context "when parent class has default scope" do 
+      
+      before do 
+        Sound.send(:include, Ownable)
```

Comment:
Why is this include here and not in the model definition itself?

Reply:
A test should not be mutating model classes. Either this test needs a new model class or one of the existing defintions needs to be changed in a way that doesn't break any of the other tests.

Reply:
I was actually just following an example [here](https://github.com/mongodb/mongoid/blob/5c90efb00347e25d8b5ff0ce27d00ac8c906f0e2/spec/mongoid/fields_spec.rb#L1198) that does the same thing. This is done to allow for access of the subclass that lives in the same file without giving it its own file. I can just put the subclass in its own file instead, no big deal.

Reply:
done

Reply:
I see, this is one of the things in the existing code that we shouldn't be doing going forward.